airgun/browser.py

```diff
@@ -46,11 +46,11 @@ def _sauce_ondemand_url(saucelabs_user, saucelabs_key):
     :param str saucelabs_key: saucelabs access key
     :return: string representing saucelabs ondemand URL
     """
-    return 'http://{0}:{1}@ondemand.saucelabs.com:80/wd/hub'.format(
+    return 'http://{}:{}@ondemand.saucelabs.com:80/wd/hub'.format(
```

Comment:
is this an example of needing a manual f-string conversion?

Reply:
correct, and should be updated in the latest commit, I manually went through and updated _almost_ all of them.